dao.py

```diff
@@ -559,7 +559,7 @@ def insert_user(self, user):
 
     # throws invalidRegionsException, which is okay, as this is only used by a
```

Comment:
let's remove this comment, it's no longer true with the introduction of the admin dashboard.